mesheryctl/internal/cli/root/system/check.go

```diff
@@ -271,12 +274,14 @@ func (hc *HealthChecker) runKubernetesAPIHealthCheck() error {
 	podInterface := client.KubeClient.CoreV1().Pods("")
 	_, err = podInterface.List(context.TODO(), v1.ListOptions{})
 	if err != nil {
+		if hc.context.Platform == "kubernetes" { // increase failure count
+			failure++
```

Comment:
what is the use of incrementing failures count here, if we return eventually at line:284?

Reply:
it's just too hard to explain how these checks work it would be easier to explain in a meeting, but we need to increment the count of failure.

Reply:
Granted, we are trying to get this fix through quickly, if the [design spec](https://docs.google.com/document/d/1WUbupfwrnzKlmh_T5eCkDjMfsVxlGgAwaLa46t3Kn_0/edit) doesn’t describe this behavior, let’s convert this comment to an issue for follow up post merge of the pr.